src/org/opensolaris/opengrok/search/context/QueryMatchers.java

```diff
@@ -89,9 +90,19 @@ private void getTerms(Query query) {
             getTerm((TermQuery) query);
         } else if (query instanceof PrefixQuery) {
             getPrefix((PrefixQuery) query);
+        } else if (query instanceof RegexpQuery) {
```

Comment:
Similar 'GetTerm' can be found in Summarizer.java,
what exactly does Summarizer? Does it makes sense to add RegexpQuery there too?